conf/nginx/conf.d/prod.conf

```diff
@@ -0,0 +1,44 @@
+server {
+        listen 80;
+        server_name localhost;
+
+        location / {
+                proxy_pass http://tomcat_oscar:8080;
+                proxy_set_header Host $host;
+                proxy_set_header X-Real-IP $remote_addr;
+                proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+                proxy_http_version 1.1;
+                proxy_set_header Upgrade $http_upgrade;
+                proxy_set_header Connection "upgrade";
+
+                sub_filter_types text/css;
+
+                sub_filter 'OSCAR McMaster Trunk' '<div id="login-title">${LOGIN_TITLE}</div>';
+                sub_filter 'images/OSCAR-LOGO.gif' '/static/images/OSCAR-LOGO.png';
+                sub_filter 'img src' 'img id="login-logo" src';
+                sub_filter 'build.dateTime' '${BUILD_DATE}';
+                sub_filter 'build.JOB_NAME' '${BUILD_NAME}';
+                sub_filter 'build.BUIL_DNUMBER' '${BUILD_NUMBER}';
+                sub_filter '<font' '<p>${LOGIN_TEXT}</p><font';
+                sub_filter '</head>' '<link rel="stylesheet" href="/static/css/oscar-custom.css" /></head>';
+                sub_filter_once off;
+        }
+
+        location /static {
+                alias /static;
+                autoindex on;
+                expires 30d;
+        }
+
+        error_log  /var/log/nginx/error.log  warn;
+
+}
+
+server {
+        listen 443 ssl;
+        
+        server_name openosp.ca;
+        
+        ssl_certificate      /ssl/certww.pem;
+        ssl_certificate_key  /ssl/privkey.pem;
```

Comment:
@ferdinandpolpol we still want everything (oscar) to work on https, so don't we need to proxy it to port 80, or repeat/include the same location code?